docs/MainSite/Installation.md

```diff
@@ -173,6 +173,10 @@ or (equivalently) use the `module load` command.
 it is recommended you read the [documentation](https://spack.readthedocs.io) if
 you require features such as packages installed with different compilers.
 
+**Note**: On a Mac, you may need to `spack install
+yaml-cpp@develop~shared` in order to force the building of the static
```

Comment:
It's probably worth noting `(tilde not dash)` since with my font at least it looks like a dash.